### ISSUE TITLE
Hello.

### ISSUE THREAD

**Issue by GaClik** [mdciri/Vox2Vox#3]
Hello,
Thanks for sharing your code. I'm working on the BRats2018 data set. I ran your code 220 epochs. But I couldn't find the same results as you. My Test results:
Evaluation 57/57 (100.00%) [0.75832042 0.47969846 0.46253171]
[34.80947115 25.43625307 22.91565778].

Unlike your code, I manually separated the datasets.
and class weights:
class_weights = [0.25646058, 59.627304, 16.495895, 42.784115]
class_weights = numpy.array (class_weights)

I couldn't figure out where I made a mistake. I would be very happy if you could help me with this subject.

**Comment by mdciri** [comment]
are you saving the weights every time the validation loss decreases or do you use the model trained with the weights after 220 epochs?
